assets/pages/bundles-landing/reducers/reducers.js

```diff
@@ -5,6 +5,7 @@
 import { combineReducers } from 'redux';
 
 import { abTestReducer as abTests } from 'helpers/abtest';
+import { intCmpReducer as intCmp } from 'helpers/intCmp';
```

Comment:
Personally, I prefer to export objects with functions inside. In this way, it is flexible enough to allow us to export an arbitrary amount of functions and always treat the imports in the same way. For example, if we develop a module which only exports one function and in the future, the module iterated and has more exports, then we should modify all the imports, treat them as an object will let us avoid those modification. 
Unfortunately, we currently have an *eslint* rule which enforces that in the case of just one export we should export default a function. My plan is to soon get rid of this rule in order to be able to standardise all the files. 

Link to the documentation [here](https://developer.mozilla.org/en/docs/web/javascript/reference/statements/export)